pavelib/quality.py

```diff
@@ -258,6 +264,16 @@ def run_quality(options):
     pep8_reports = u' '.join(pep8_files)
 
     try:
+        violations_str = None
+        with open(pep8_reports, 'r') as f:
+            violations_list = f.readlines()
+            violations_str = '\n'.join(violations_list)
+
+        if violations_str:
+            raise BuildFailure(
```

Comment:
This is causing the build to fail before anything is written to the diff-quality directory, which is why those reports don't show up. See [is_percentage_failure](https://github.com/edx/edx-platform/blob/sarina/pep8-quality-report/pavelib/quality.py#L327) method.